src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,12 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+            .flatMap(n -> Arrays.stream(n.split(",")))
+            .mapToInt(n -> Integer.parseInt(n))
```

Comment:
reduce number of `map` operations as much as possible
```suggestion
            .flatMap(n -> Arrays.stream(n.split(",")))
            .mapToInt(n -> Integer.parseInt(n))
```

Reply:
I'm not sure that this solution is ok, but I can't understand how I can reduce it in another way:
            .flatMapToInt(n -> Arrays.stream(n.split(","))
                .mapToInt(Integer::parseInt))